infrastructure/laa-estimate-financial-eligibility-for-legal-aid-uat/permanent-redirects.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: check-client-qualifies-laa-estimate-eligibility-redirects
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-redirects-laa-estimate-financial-eligibility-for-legal-aid-uat-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/permanent-redirect: "https://check-your-client-qualifies-for-legal-aid.service.gov.uk"
+spec:
+  rules:
+    # Add rules for every host that you would like to redirect to production, like so:
+    - host: redirect-example-check-client-qualifies.cloud-platform.service.justice.gov.uk
```

Comment:
It would be nice if the same rule could handle multiple hosts or that we at least didn't have to repeat all these lines, but looks like that's not possible, is it?

Reply:
Interestingly (to me), when I had this file in /helm_deploy/templates alongside ingress.yaml, a bunch of validation occurred that insisted I needed to add a bunch of guff for each rule I added. But it turns out that now that this is in `infrastructure/` and gets applied in CI by a simple `kubectl apply -f`, that validation is skipped and I can get away guff-free.

Reply:
Ah I hadn't twigged that this wasn't in helm. Neat! It absolutely doesn't need to be. (although I do slightly anticipate a small amount of confusion of where this other ingress came from that isn't adjacent in the code 😅)

I'm surprised helm didn't like it anyway, but hey